playwright.config.ts

```diff
@@ -25,7 +25,7 @@ export default defineConfig({
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: 'http://127.0.0.1:3000',
+    baseURL: 'http://localhost:5173',
```

Comment:
suggestion - keeping the old port will be safer
https://vitejs.dev/config/server-options.html#server-port

Reply:
Keeping the old port gave me an error when I was trying to run the tests locally.

Reply:
Also, this is mentioned in the docs -  "if the port is already being used, Vite will automatically try the next available port so this may not be the actual port the server ends up listening on."

Reply:
Test failed once again🤔 - https://github.com/hasadna/open-bus-map-search/pull/86/checks